_data/sidebars/sidebar.yml

```diff
@@ -14,11 +14,7 @@ entries:
       type: homepage
       output: web
 
-    - title: Change Log
-      url: /changelog
-      output: web
-
-    - title: Introduction
+    - title: Usage example
```

Comment:
I think we should keep a page called **Introduction**, but I also like your naming **Usage example**, or maybe **GraphWalker Use Cases**. Instead of changing the name of this page, I suggest that we create a new one with the use cases instead.